### DISCUSSION TITLE
Why wasn't my alcohol fuel ignited

### DISCUSSION THREAD

**Discussion by hunzideriji** [firemodels/fds#13460] [Q&A]
According to the instructions in the user manual, I set up an alcohol fuel model, but my alcohol was not ignited. May I ask why this is happening
[新建文本文档 (2).txt](https://github.com/user-attachments/files/17096006/2.txt)

**Comment by drjfloyd** [comment]
EPUMO2=1.31E+8

You are a factor of 10,000 high for this value.

**Reply by hunzideriji** [reply]
> EPUMO2=1.31E+8
> 
> 对于此值，您是 10,000 的高系数。

Thank you for your answer. We found that the default value for EPUMO2 is 1.3e4. However, after correcting it, it is shown that the flame temperature was around 70 ℃, which is clearly lower than the experimental results. Do you think a chemical reaction has occurred?

**Reply by drjfloyd** [reply]
If I run your case after fixing the EPUMO2 I see this for the x drection temperature slice.
![test_0088](https://github.com/user-attachments/assets/daabf55f-8131-479b-8414-27f7bb93a270)

**Reply by hunzideriji** [reply]
> 如果我在修复 EPUMO2 后运行您的案例，我会看到 x 勃起温度切片。 ![test_0088](https://private-user-images.githubusercontent.com/12799217/369849026-daabf55f-8131-479b-8414-27f7bb93a270.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcwODk5MzEsIm5iZiI6MTcyNzA4OTYzMSwicGF0aCI6Ii8xMjc5OTIxNy8zNjk4NDkwMjYtZGFhYmY1NWYtODEzMS00NzliLTg0MTQtMjdmN2JiOTNhMjcwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIzVDExMDcxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEwNTU5YzUwMDM4YTA2OTk0YmIzMzAzMzk1YWU0YzUzMzU4ZDBhZWZkYjIwNjI0ZjkyMjIyZGY2MTVkOTIxZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.44Y6M0bSl_PPKlnahUkO_cSONf2dg3PCVs1UaEDwnJE)

Thank you for your answer. The igniter disappears after running for 1 second, and the temperature may suddenly drop to 70 ℃ afterwards. I don't know if your running results are like this?

**Reply by hunzideriji** [reply]
We have made more attempts, but still cannot ignite the ethanol. It can be observed that after the ignition source removed, the evaporation gas  ocurred above the liquid, and the temperature of the gas is around 70 ℃. We speculate that this may be due to the ethanol evaporates are reaching the boiling states. It is worth noting that the ignition temperature of ethanol is sufficiently low ( it should be around 70 ℃). That is to say, the fuel has already undergone a chemical reaction at this temperature. This is confusing me, I don't know if you can answer my question.

**Reply by drjfloyd** [reply]
I did not run that far out. You did not give the explanation that at first you had high temperatures that then dropped.  

1 s may not be long enough to preheat the pool of liquid.

**Reply by drjfloyd** [reply]
Another thing to consider is your red igntion block sits directly on the yellow target block. When you remove the ignition OBST, the yellow target cells underneath will be at TMPA. There is no conduction from the red to the yellow
